09-schema/03-rules.md

```diff
@@ -274,6 +274,112 @@ TypeQLDefine query = TypeQL.define(
 </div>
 
 This rule will make every relation transitive.
+## Efficient transitivity
+
+A common use-case for rules is to infer the transitive closure of a relation. The most straight-forward way of doing this is as follows:
```

Comment:
Here you should already say this is for a bound start point - `A common use-case for rules is to infer all transitively reachable concepts from a particular starting concept (consider finding all teams a particular user is a member of recursively, or all parts that make up a particular machine, etc.)`